setup.py

```diff
@@ -11,6 +11,7 @@
     'tornado>=4.2.1',
     'cryptography>=0.2.1',
     'pyopenssl',
+    'PyYAML'
```

Comment:
You probably want to specify a minimum version, https://github.com/docker/compose/blob/master/setup.py#L32 shows a good example. !resolve